docs/data_managers/using_safedata/posting_metadata.md

```diff
@@ -0,0 +1,30 @@
+# Updating the metadata server
+
+## Posting data for new published datasets
```

Comment:
Would be good to include a "what if I don't have a metadata server but might have one in the future?" section somewhere